tailwind.config.js

```diff
@@ -1,41 +1,34 @@
+const plugin = require('tailwindcss/plugin');
+
 module.exports = {
   theme: {
+    /**
+     * Themeable
+     */
     borderRadius: {
       none: '0',
       1: "var(--calcite-border-radius)",
       half: '50%',
       full: '100%',
     },
     boxShadow: {
+      // assets/styles/_shadow
       '1-sm': 'var(--calcite-shadow-1-press)',
       '1': 'var(--calcite-shadow-1)',
       '1-lg': 'var(--calcite-shadow-1-hover)',
       '2-sm': 'var(--calcite-shadow-2-press)',
       '2': 'var(--calcite-shadow-2)',
       '2-lg': 'var(--calcite-shadow-2-hover)',
+      'border-bottom': 'var(--calcite-shadow-border-bottom)',
```

Comment:
Could existing instances of this be replaced with `border-b`? Is this enough of an edge case that we could remove it?

Reply:
Hmmm...this one used box-shadow.
This is to avoid conflicting with border styles, but I can have another look at how that is being done.